MdeModulePkg/Bus/Pci/NvmExpressDxe/NvmExpressHci.c

```diff
@@ -1093,6 +1093,8 @@ NvmeUnregisterShutdownNotification (
   EFI_STATUS                       Status;
   EFI_RESET_NOTIFICATION_PROTOCOL  *ResetNotify;
 
+  ReportStatusCode ((EFI_ERROR_MAJOR | EFI_ERROR_CODE), (EFI_IO_BUS_SCSI | EFI_IOB_EC_INTERFACE_ERROR));
+

```

Comment:
This function will be trigged when stop the device. It may not be time out device error.

Reply:
Thanks, I took a look at the history here and this was an from a much older version of this file. I will drop it as I agree it does not make sense here.